test/y2storage/filesystems/type_test.rb

```diff
@@ -231,4 +231,27 @@
       end
     end
   end
+
+  describe "label_valid_chars" do
```

Comment:
convention is to write "#label_valid_chars". "#" is used for instance methods and "." for class methods.

Reply:
ok.